apps/journeys/src/components/blocks/GridContainer/GridContainer.tsx

```diff
@@ -0,0 +1,33 @@
+import { ReactElement } from 'react'
+import { GetJourney_journey_blocks_GridContainerBlock as GridContainerBlock } from '../../../../__generated__/GetJourney'
+import { TreeBlock } from '../../../libs/transformer/transformer'
+import { Grid as MaterialGrid, GridDirection } from '@mui/material'
+import { BlockRenderer } from '../../BlockRenderer'
+
+export function GridContainer({
+  spacing,
+  direction,
+  justifyContent,
+  alignItems,
+  children
+}: TreeBlock<GridContainerBlock>): ReactElement {
+
+  return (
+    <MaterialGrid 
+      container
+      spacing={spacing}
+      direction={hyphenate(direction) as GridDirection}
```

Comment:
Possibly don't need this now and linting needs to be fixed, but other than that - looks good :)
```suggestion
      direction={hyphenate(direction)}
```

Reply:
From Slack:

![image](https://user-images.githubusercontent.com/10802634/139745366-157d23b8-6fa8-4198-8e8b-e28942e1fac3.png)